agents/base.py

```diff
@@ -8,5 +11,13 @@ class BaseAgent(metaclass=MetaParent):
 
 
 class TorchAgent(BaseAgent, nn.Module):
+    @abstractmethod
+    def forward(self, inputs: Dict[str, Any]):
+        pass
```

Comment:
Let's add `raise NotImplementedError` instead of `pass`
```suggestion
        raise NotImplementedError
```